internal/scorecard/alpha/tests/test_bundle.go

```diff
@@ -43,50 +39,15 @@ func GetBundle(bundlePath string) (cfg TestBundle, err error) {
 
 	// TODO evaluate another API call that would support the new
 	// bundle format
-	_, cfg.Bundles, cfg.BundleErrors = manifests.GetManifestsDir(bundlePath)
-
-	// get CRs from CSV's alm-examples annotation, assume single bundle
-	cfg.CRs = make([]unstructured.Unstructured, 0)
+	var bundles []*registry.Bundle
+	//var bundleErrors []errors.ManifestResult
+	_, bundles, _ = manifests.GetManifestsDir(bundlePath)
 
-	if len(cfg.Bundles) == 0 {
-		return cfg, fmt.Errorf("no bundle found")
-	}
-
-	csv, err := cfg.Bundles[0].ClusterServiceVersion()
+	bundle = *bundles[0]
+	_, err = bundle.ClusterServiceVersion()
 	if err != nil {
-		return cfg, fmt.Errorf("error in csv retrieval %s", err.Error())
-	}
-
-	if csv.GetAnnotations() == nil {
-		return cfg, nil
-	}
-
-	almExamples := csv.ObjectMeta.Annotations["alm-examples"]
-
-	if almExamples == "" {
-		return cfg, nil
-	}
-
-	if len(cfg.Bundles) > 0 {
-		var crInterfaces []map[string]interface{}
-		err = json.Unmarshal([]byte(almExamples), &crInterfaces)
-		if err != nil {
-			return cfg, err
-		}
-		for i := 0; i < len(crInterfaces); i++ {
-			buff := new(bytes.Buffer)
-			enc := json.NewEncoder(buff)
-			err := enc.Encode(crInterfaces[i])
-			if err != nil {
-				return cfg, err
-			}
-			obj := &unstructured.Unstructured{}
-			if err := obj.UnmarshalJSON(buff.Bytes()); err != nil {
-				return cfg, err
-			}
-			cfg.CRs = append(cfg.CRs, *obj)
-		}
+		return bundle, fmt.Errorf("error in csv retrieval %s", err.Error())
 	}
 
-	return cfg, err
+	return bundle, err
```

Comment:
Nit: since we just checked the error
```suggestion
	return bundle, nil
```

Reply:
change made.